### PR TITLE
feat: 그룹 내보내기 기능 추가

### PR DESCRIPTION
**Pull request by myeongjunkim**
image

# 체크리스트

- [x]  그룹 리스트 추가
- [x]  그룹장 태그 추가
- [x]  그룹 내보내기 기능 구현

### CODE REVIEW

**Comment by myeongjunkim** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/TeamVisioneer/PrayU-Web/240?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* **#240** <a href="https://app.graphite.dev/github/pr/TeamVisioneer/PrayU-Web/240?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#239** <a href="https://app.graphite.dev/github/pr/TeamVisioneer/PrayU-Web/239?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#237** <a href="https://app.graphite.dev/github/pr/TeamVisioneer/PrayU-Web/237?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#236** <a href="https://app.graphite.dev/github/pr/TeamVisioneer/PrayU-Web/236?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#235** <a href="https://app.graphite.dev/github/pr/TeamVisioneer/PrayU-Web/235?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @myeongjunkim and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: src/components/group/GroupSettingsDialog.tsx [hunk 1]
```diff
@@ -41,6 +54,23 @@ const GroupSettingsDialog: React.FC<GroupSettingsDialogProps> = ({
...
+      onAction: async () => {
+        await deleteMemberbyGroupId(member.profiles.id, targetGroup.id);
+        await deletePrayCardByGroupId(member.profiles.id, targetGroup.id);
+        window.location.replace(`/group/${targetGroup.id}`);
+        analyticsTrack("클릭_그룹_내보내기", { group_id: targetGroup.id });
+      },
+    });
+    setIsConfirmAlertOpen(true);
+  };
+
```
Comment (myeongjunkim) [inline src/components/group/GroupSettingsDialog.tsx:73]: 워딩 괜찮은지 점검 부탁드립니다!

File: src/components/todayPray/TodayPrayCardUI.tsx [hunk 2]
```diff
@@ -23,18 +23,16 @@ const PrayCardUI: React.FC<PrayCardProps> = ({ prayCard, eventOption }) => {
...
 
   return (
     <div className="flex flex-col gap-2 min-h-80vh max-h-80vh">
       <div className="flex justify-between px-2">
         <div className="w-6"></div>
         <div className="text-sm text-gray-400">
-          {prayCardCarouselApi &&
-            prayCardCarouselApi?.scrollSnapList().length - 2}
-          명 중 {prayCardCarouselIndex}
+          {prayCardCarouselList.length}명 중 {prayCardCarouselIndex}
```
Comment (myeongjunkim) [inline src/components/todayPray/TodayPrayCardUI.tsx:35]: 버그가 있어 수정합니다.

File: src/pages/TutorialPage.tsx [hunk 3]
```diff
@@ -201,8 +202,15 @@ const TutorialPage: React.FC = () => {
...
-    <div className="flex justify-center gap-[30px]">
+  const ReactionBtn = (
+    <div className="relative flex justify-center gap-[30px]">
+      {!todayPrayType && (
+        <TbHandClick
+          size={30}
+          className="absolute z-50 bottom-0 right-24 animate-pulse duration-1000 ease-in"
+        />
+      )}
+
```
Comment (myeongjunkim) [inline src/pages/TutorialPage.tsx:213]: image

본 PR 에서 튜토리얼 클릭버튼 관련 간단 수정도 같이 하고 넘어갑니다.

File: src/pages/StoryPage/StoryPage.tsx [hunk 4]
```diff
@@ -3,7 +3,7 @@ import MainHeader from "../MainPage/MainHeader";
 
 const StoryPage = () => {
   return (
-    <div className="flex flex-grow flex-col justify-center text-center w-full items-center bg-white ">
+    <div className="flex flex-grow flex-col justify-center text-center w-full items-center bg-white pt-11">
```
Comment (myeongjunkim) [inline src/pages/StoryPage/StoryPage.tsx:6]: image

상단이 짤리고 있어서 수정합니다.

File: src/components/group/GroupSettingsDialog.tsx (hunk 1 above)
Comment (yaminsu5674) [reply src/components/group/GroupSettingsDialog.tsx:73]: '요'자로 끝나도 좋을 것 같습니다!
ex. 내보시겠어요? ,  기도카드는 모두 삭제되어요:(, ... 처럼요!

File: src/pages/StoryPage/StoryPage.tsx (hunk 4 above)
Comment (yaminsu5674) [reply src/pages/StoryPage/StoryPage.tsx:6]: 고맙습니다 ㅎㅎ

**Review by yaminsu5674 — APPROVED** [review]
고생많으셨습니다!

**Review by treephesians — APPROVED** [review]
감사합니다! 고생하셨습니다 👍🏻

**Pull request merged**